endpoints/citrus-camel/src/main/java/com/consol/citrus/camel/actions/StartCamelRouteAction.java

```diff
@@ -45,7 +45,7 @@ public void doExecute(TestContext context) {
 
             try {
                 ((AbstractCamelContext) camelContext).startRoute(route);
-                log.info(String.format("Started Camel route '%s' on context '%s'", route, camelContext.getName()));
+                LOG.info(String.format("Started Camel route '%s' on context '%s'", route, camelContext.getName()));
```

Comment:
same discussion as on last PR, would use parameter replacement by slf4j.
honestly, the chance that we'll ever change the logger implementation is.. quit low? :D

Reply:
yeah probably. let's do this in a separate issue to tackle all of them at once, ok?

Reply:
agree 👍🏼

Reply:
we actually have #829. I have added a comment.